plugin/pxr/maya/lib/usdMaya/testenv/testUsdExportPointInstancer.py

```diff
@@ -52,9 +52,17 @@ def tearDownClass(cls):
     def setUpClass(cls):
         standalone.initialize('usd')
 
-        cmds.loadPlugin("MASH")
-
-        scene = "InstancerTestMash.ma"
+        # Choose the test file based on whether the MASH plugin is available.
```

Comment:
Again, incorrect revert of
https://github.com/Autodesk/maya-usd/commit/7cd3c48e173434ec9cdb074f7a2adfade4fd0258
will fix.